network-config-analyzer/NetworkConfig.py

```diff
@@ -274,7 +294,7 @@ def get_affected_pods(self, is_ingress):
         :rtype: Peer.PeerSet
         """
         affected_pods = Peer.PeerSet()
-        for policy in self.sorted_policies:
+        for policy in self.sorted_policies + self.ingress_allow_policies + self.ingress_deny_policies:
```

Comment:
Same question here - Do we want Ingress policies to be considered here or not?

Reply:
Same as above